mappings/net/minecraft/block/Block.mapping

```diff
@@ -105,8 +105,10 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 		ARG 1 world
 		ARG 2 pos
 		ARG 3 side
-	METHOD method_22358 (Lnet/minecraft/class_2680;Lnet/minecraft/class_3611;)Z
+	METHOD method_22358 canBucketPlace (Lnet/minecraft/class_2680;Lnet/minecraft/class_3611;)Z
```

Comment:
Is this related to `FluidFillable.canFillWithFluid`?

Reply:
No, this is only called by BucketItem when the bucket tries to place a fluid on the block. This is overridden by end portal and end portal frames to prevent bucket fluid placement.

Reply:
Shouldn't this be `canPlaceBucket`? Or even better, `canEmptyBucket`

Reply:
Note the block state and fluid parameters, so I called it `canBucketPlace(Fluid)`